engine/src/flutter/lib/ui/fixtures/shaders/general_shaders/circle_sdf.frag

```diff
@@ -7,13 +7,12 @@
 #include <flutter/runtime_effect.glsl>
 
 out vec4 fragColor;
-
 uniform vec2 uSize;
-uniform sampler2D uTex;
 
 void main() {
-    vec2 p = FlutterFragCoord().xy / uSize;
-    float d = texture(uTex, p).r;
-    vec3 col = d > 0.0 ? vec3(0.0) : vec3(1.0);
-    fragColor = vec4(col, 1.0);
+  vec2 p = FlutterFragCoord().xy / uSize;
```

Comment:
Am I reading this right? It determines a distance based on whether the point is less than or greater than half the distance to the center of a box the size of uSize? If so, can we add a simple comment to that effect?

Reply:
Yea, we are drawing square textures so they are circles but they would be ovals for rectangle textures.  It's just something simple to see the results.